### PR TITLE
Add environment var

### PR DESCRIPTION
**Pull request by AndyJiang99**

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -7,9 +7,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.11.0
+version: 0.13.0
```
Comment (bruce-au) [inline helm/Chart.yaml:10]: Could we set this to a single minor version bump?

File: helm/Chart.yaml [hunk 2]
```diff
@@ -7,9 +7,9 @@ type: application
...
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: latest
+appVersion: 0.25.0
```
Comment (bruce-au) [inline helm/Chart.yaml:15]: Could we set this value to the latest release of the On Premise Agent? It should be version `0.27.0` as of writing.

**Review by bruce-au — CHANGES_REQUESTED** [review]

**Comment by bruce-au** [comment]
Looks good, just a few minor changes. Thanks for your contribution!

File: helm/templates/pod_disruption_budget.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{{- if .Values.podDisruptionBudget.enabled -}}
```
Comment (taha-au) [inline helm/templates/pod_disruption_budget.yaml:1]: let's add this to the values file

```
podDisruptionBudget:
  enabled: false
```

**Review by taha-au — CHANGES_REQUESTED** [review]

File: helm/Chart.yaml (hunk 1 above)
Comment (AndyJiang99) [reply helm/Chart.yaml]: I set this to 0.12.0 in both this PR and in the #13 PR as well so both changes need to be in together prior to deploying the 0.12.0 chart version

**Review by bruce-au — APPROVED** [review]

File: helm/templates/pod_disruption_budget.yaml (hunk 3 above)
Comment (AndyJiang99) [reply helm/templates/pod_disruption_budget.yaml]: Was added in #13, so I just updated this PR to have those new changes as well

**Comment by bruce-au** [comment]
Looks good, will merge after this PR has been rebased

**Pull request merged**